Improve some EE documentation (#606)
kestra-io/docs: content/docs/04.user-interface-guide/10-administration.md

![Kestra User Interface Home Page](/docs/user-interface-guide/20-EE-Groups-Access.png)

> A group is made of users. It can be useful to define specific group of people (mapping teams, users working on the same project, etc.).

Reviewer: ```suggestion
It's a collection of users who require the same set of permissions. It's useful to assign the same permissions to multiple users who belong to the same team or project. 
```

![Kestra User Interface Home Page](/docs/user-interface-guide/22-EE-Roles-Create.png)

> Roles allow to manage CRUD (CREATE, READ, UPDATE, DELETE) access to the different Kestra resources. They can be attached to Groups or Users.

Reviewer: ```suggestion
Roles manage CRUD (CREATE, READ, UPDATE, DELETE) access to Kestra resources such as flows, executions, or secrets. They can be attached to groups or users.
```
